WordPressKit/Time-based data/AuthorsStatsType.swift

```diff
@@ -83,6 +94,20 @@ extension StatsTopPost {
         self.postID = postID
         self.viewsCount = viewsCount
         self.postURL = URL(string: postURL)
+        self.kind = type(of: self).kind(from: jsonDictionary["type"] as? String)
+    }
+
+    static func kind(from kindString: String?) -> Kind {
+        switch kindString {
+        case "post"?:
```

Comment:
Are these `?` necessary? I removed them and the tests passed. Seems like they're not needed...

Reply:
Huh, thanks for that — I would have sworn I tried before and it failed to build before!